debian/rules

```diff
@@ -117,6 +117,7 @@ install: build
 	mv debian/temp/lib/opennms-snmp-asset-provisioning-adapter*.jar          debian/opennms-plugin-provisioning-snmp-asset/usr/share/java/opennms/
 	mv debian/temp/lib/opennms-wsman-asset-provisioning-adapter*.jar         debian/opennms-plugin-provisioning-snmp-asset/usr/share/java/opennms/
 	mv debian/temp/lib/opennms-snmp-metadata-provisioning-adapter*.jar       debian/opennms-plugin-provisioning-snmp-asset/usr/share/java/opennms/
+	mv debian/temp/lib/opennms-geoip-provisioning-adapter*.jar               debian/opennms-plugin-provisioning-snmp-asset/usr/share/java/opennms/
```

Comment:
I'm not sure anything about this plugin has anything to do with "SNMP assets" -- shouldn't it go in a different package?

Reply:
> In this specific case, you have to set in the configuration `public-ipv4` or `primary` to avoid a lookup against private ULA IPv6 addresses.

Added a statement to identify fc00::/7 as local addresses. Should work now.